lms/static/js/certificates/views/certificate_whitelist.js

```diff
@@ -54,7 +54,9 @@
                         model.destroy(
                             {
                                 success: function() {
-                                    self.showMessage('Student Removed from certificate white list successfully.');
+                                    self.showMessage(
+                                        gettext('Student Removed from certificate white list successfully.')
```

Comment:
You need to wrap these (see below) with `_.escape()` as well.  An alternative that would be safer would be to rename `showMessage()` to `escapeAndShowMessage()` and have the `_.escape()` call once inside there.  

The issue would be if the message itself required html (e.g. `"Here is an <strong>html message</strong> that can't be escaped."`).  I don't think you have that concern here,  and if you did, you could have a separate method named `showHtmlMessage()`.